src/projetos/CadastroProjetosPage.tsx

```
@@ -1,395 +1,541 @@
-                                                </FormControl>
-                                                <FormMessage className='text-red-600' />
-                                            </FormItem>
-                                        )}
-                                    />
-                                </div>
+              <div className="space-y-6">
+                {/* Imagem do projeto */}
+                <FormField
+                  control={form.control}
+                  name="imagem"
+                  render={({ field: { onChange, value, ...fieldProps } }) => (
```

ESLint

[error] 'value' is defined but never used.

(@typescript-eslint/no-unused-vars)